deployment/ccip/changeset/initial_deploy_test.go

```diff
@@ -21,61 +15,11 @@ import (
 
 func TestInitialDeploy(t *testing.T) {
```

Comment:
should remove this test and have an initial_add_chain_test.go which covers the add chain functionality

Reply:
I did not remove it because this is the only test chainlink-ccip runs

Reply:
I see ok, we can come back to it then